Channels/Insider/release_notes.md

```diff
@@ -1,8 +1,8 @@
-## October 1, 2020 Insider Release ([v1.1.1369.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.1369.01))
+## January 7, 2021 Insider Release ([v1.1.1467.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.1467.01))
```

Comment:
@adarshrema, need your approval here. You can view the customer-facing markdown at https://github.com/microsoft/accessibility-insights-windows/blob/e73e4f871ef3fccf9a6e3d8b0cdf5c8fae35a125/Channels/Insider/release_notes.md